From 33f66c2af5d53d240be08f4f8a9dc190bdb42e15 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 26 Nov 2024 23:15:40 +0100
Subject: [PATCH] [doged] Rebrand to Dogecash to doged (#86)

Part of an ongoing rebrand effort. See #82.

Current-ABC adaptation (2026-10-14): Identify current CMake package and installer label as doged, retaining upstream version and packaging machinery rather than importing obsolete build configuration.

Original-Commit: 34ab40c4a0babb49ec0f85259ee544220a0427a6

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fb513579ec55b8e4c1c3e2666b4b947968b417e..5a4cf2aa92495379568b6186382de7424a576d7f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 
 project(bitcoin-abc
 	VERSION 0.34.1
-	DESCRIPTION "Bitcoin ABC is a full node implementation of the eCash protocol."
+	DESCRIPTION "doged is a full node implementation of the Dogecoin protocol."
 	HOMEPAGE_URL "https://www.bitcoinabc.org"
 )
 
@@ -23,7 +23,7 @@ add_custom_target(print-project-name
 )
 
 # Package information
-set(PACKAGE_NAME "Bitcoin ABC")
+set(PACKAGE_NAME "doged")
 set(PACKAGE_BUGREPORT "https://github.com/Bitcoin-ABC/bitcoin-abc/issues")
 
 # Copyright
diff --git a/PackageOptions.cmake b/PackageOptions.cmake
index e17737e0b8bda0e3b3241b4233cd0b0c2d94f611..a1df82c250858a5b866aadb6f9b7e133c009c21c 100644
--- a/PackageOptions.cmake
+++ b/PackageOptions.cmake
@@ -57,7 +57,7 @@ macro(add_start_menu_link LINK_NAME EXE PARAMETERS ICON_EXE ICON_INDEX)
 	)
 endmacro()
 
-set(CPACK_NSIS_MENU_LINKS "${CMAKE_INSTALL_BINDIR}/${_nsis_bitcoin_qt}" "Bitcoin ABC")
+set(CPACK_NSIS_MENU_LINKS "${CMAKE_INSTALL_BINDIR}/${_nsis_bitcoin_qt}" "doged")
 add_start_menu_link("${PACKAGE_NAME} (testnet)"
 	"${_nsis_bitcoin_qt}"
 	"-testnet"
